<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactsWindow</class>
 <widget class="QDialog" name="ContactsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>GOST type A</family>
       <pointsize>14</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Users</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="usersList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="disconnectButton">
       <property name="font">
        <font>
         <family>GOST type A</family>
         <pointsize>14</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Disconnected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="banButton">
       <property name="font">
        <font>
         <family>GOST type A</family>
         <pointsize>14</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Banned</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="unbanButton">
       <property name="font">
        <font>
         <family>GOST type A</family>
         <pointsize>14</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Disbanned</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
